### PR TITLE
Hackytaq replicates

### PR DESCRIPTION
**Pull request by ericmjl**

### CODE REVIEW

File: experiments/20161006-hackytaq-replicates/20161006-hackytaq-replicate.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+## Goals
+
+Following up on Vivian's set of excellently-conducted prior experiments (date: [2016-09-20](../20160920-Taq-Mn-mutagenesis/20160912-mgcl2-taq-series.md)), I wanted to fill in the blanks w.r.t. the concentrations not yet tested and sequenced.
```
Comment (Starnite) [inline experiments/20161006-hackytaq-replicates/20161006-hackytaq-replicate.md]: What does "fill in the blanks" signify?

**Review by Starnite — CHANGES_REQUESTED** [review]

File: experiments/20161006-hackytaq-replicates/20161006-hackytaq-replicate.md (hunk 1 above)
Comment (ericmjl) [reply experiments/20161006-hackytaq-replicates/20161006-hackytaq-replicate.md]: "fill in the blanks" refers to testing the Manganese concentrations not sent for sequencing in your previous run ([note](https://github.com/ericmjl/protein-systematic-characterization/blob/master/experiments/20160920-Taq-Mn-mutagenesis/20160912-mgcl2-taq-series.md)). I will update this paragraph explicitly.

Comment (ericmjl) [reply experiments/20161006-hackytaq-replicates/20161006-hackytaq-replicate.md]: Also, some concentrations were repeated to try to get a sense of the variation in the num of mutations compared to your experimental run. As we see, of course, the 0 mM control had an unusually high mutation rate, and so I think all of the data isn't good.

Comment (ericmjl) [reply experiments/20161006-hackytaq-replicates/20161006-hackytaq-replicate.md]: You may want to update the note with the results from sending in 0.15, 0.20, 0.30 and 0.35 mM  Mn<sup>2+</sup> for sequencing.

**Review by Starnite — COMMENTED** [review]
LGTM! I'll make the requested update too.

**Review by Starnite — APPROVED** [review]

**Comment by Starnite** [comment]
@ericmjl Don't forget to delete this branch!

**Pull request merged**